### PR TITLE
API changes - Account Roles

### PR DESCRIPTION
**Pull request by kris-daxiom**
*Issue:*

- bcgov/entity/issues/23023

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Review by dimak1 — APPROVED** [review]
LG!

File: strr-api/src/strr_api/resources/application.py [hunk 1]
```diff
@@ -111,6 +112,9 @@ def create_application():
         if not valid:
             return error_response(message="Invalid request", http_status=HTTPStatus.BAD_REQUEST, errors=errors)
         validate_request(json_input)
+        roles = AccountService.list_account_roles(account_id=account_id)
```
Comment (shaangill025) [inline strr-api/src/strr_api/resources/application.py:115]: How do you plan on handling platform related roles in future? Based on application type?

**Review by shaangill025 — APPROVED** [review]
LGTM

**Comment by rstens** [comment]
Causes the current UI Code to malfunction.

**Comment by rstens** [comment]
Changes should have been synced or at least very close together.

**Pull request merged**